src/Internal/Payment/State/InitialState.php

```diff
@@ -78,37 +78,42 @@ public function __construct(
 	}
 
 	/**
-	 * Initialtes the payment process.
+	 * Initiates the payment process.
+	 *
+	 * @param PaymentRequest $request The incoming payment processing request.
 	 *
-	 * @param PaymentRequest $request    The incoming payment processing request.
-	 * @return CompletedState            The next state.
+	 * @return AbstractPaymentState      The next state.
 	 * @throws StateTransitionException  In case the completed state could not be initialized.
 	 * @throws ContainerException        When the dependency container cannot instantiate the state.
 	 * @throws Order_Not_Found_Exception Order could not be found.
 	 * @throws PaymentRequestException   When data is not available or invalid.
 	 */
-	public function process( PaymentRequest $request ) {
-		$context  = $this->get_context();
-		$order_id = $context->get_order_id();
-
+	public function start_processing( PaymentRequest $request ) {
 		// Populate basic details from the request.
 		$this->populate_context_from_request( $request );
 
 		// Populate further details from the order.
 		$this->populate_context_from_order();
 
+		$context = $this->get_context();
```

Comment:
Now that the context is only used for creating the intent, let's move this line within the `try {}` block.